broadcasting.md

```diff
@@ -247,6 +255,47 @@ npm run dev
 > **Note**  
 > To learn more about compiling your application's JavaScript assets, please consult the documentation on [Vite](/docs/{{version}}/vite).
 
+Additional supported features :
+
+*1. Modify private/presence channel capability. Default: Full capability*
+- Channel access control rights are granted for each individual user separately using `ably-capability`.
+- It defines list of access claims as per [Channel Capabilities](https://ably.com/docs/core-features/authentication#capability-operations).
+
+```php
+  // file - routes/channels.php
+  // User authentication is allowed for private/presence channel returning truthy values and denied for falsy values.
+  
+  // for private channel
+  Broadcast::channel('channel1', function ($user) {
+      return ['ably-capability' => ["subscribe", "history"]];
+  });
+  
+  // for presence channel
```

Comment:
```suggestion
  // for presence channels
```

Reply:
Again it's just a single channel named `channel2`
` Broadcast::channel('channel2', function ($user) {`

Reply:
see https://github.com/ably-forks/laravel-docs/pull/6#discussion_r1087868859

Reply:
done